crypto/doc/CHANGELOG_oberon.rst

```diff
@@ -9,6 +9,105 @@ Changelog - nrf_oberon
 
 All notable changes to this project are documented in this file.
 
+nrf_oberon - 3.0.14
+*******************
+
+New version of the nrf_oberon library with the following changes.
+
+Added
+=====
+
+* Added Oberon PSA core, a heavily code-size optimized and efficient implementation of PSA core licensed for use on Nordic Semiconductor devices.
```

Comment:
nrfxlib is no longer the location for the Oberon PSA core, so this line seems a bit out-of-context... 

I recommend changing to state something about the crypto deliverables from nrf_oberon instead of mentioning Oberon PSA core. efficiency and license is still true

Reply:
Done